imap_processing/spice/geometry.py

```diff
@@ -136,7 +143,7 @@ def get_spacecraft_spin_phase(
 
```

Comment:
Tim had brought up a good point in a meeting which is: what would happen if a spin failed to be recorded in the .csv. I don't immediately see how this would be handled here, but I might just be missing it.

Reply:
Laura, I believe this check `invalid_spin_phase_range = (spin_phases < 0) | (spin_phases > 1)` will capture for this. If a spin data is missing, it will result in spin phase being less than or greater than 1. @subagonsouth correct me if that's not case.

Reply:
Yes, checking that the spin_phase is in the range [0, 1) is a good way to do this check, IMO.

Reply:
I pushed changes to correct what I had to `[0, 1)`.